CONTRIBUTING.md

```diff
@@ -47,47 +47,48 @@ First, ensure that [Yarn](https://yarnpkg.com/) and
 
 ## The demo project
 
-The [demo project](demo) was initially forked from
-[dvc-checkpoints-mnist](https://github.com/iterative/dvc-checkpoints-mnist/tree/make_checkpoint),
-and is provided as a lightweight, convenient testbed to try the extension out
-with. It is not an exhaustive showcase of DVC's features, so testers are
-encouraged to try other DVC repositories -- especially real-world cases!
+The [demo project](demo) is provided as a lightweight, convenient testbed to try
+the extension out with. It is not an exhaustive showcase of DVC's features, so
+testers are encouraged to try other DVC repositories -- especially real-world
+cases!
 
-- Go to the `demo/` directory of this repo.
+- Install [Yarn](https://yarnpkg.com/) if needed. Run `yarn setup:venv` from the
+  root of this repo to set up a virtual Python environment for the demo project
+  (in `demo/.env`).
 
-- It is recommended that you create the project's virtual environment with
-  Python 3.8 as some dependencies are not yet available for Python 3.9
+- Important: Open the `demo/` folder of this repo directly in VS Code.
 
-- `yarn install` is a prerequisite for automatically setting up the demo
-  project.
+- Pull the project data with the extension (either from the **DVC Tracked**
+  panel in the _File Explorer_ or the **DVC panel** in the _Source Code_ view)
+  or by running `dvc pull` from a VS Code _Terminal_.
 
-- Run `yarn setup:venv` from the monorepo root (not `./demo`) to automatically
-  setup the Python virtual environment in the demo project.
+  > Remember to activate the virtual env first (`source .env/bin/activate`) if
```

Comment:
it should be activated automatically if you run Terminal in VS Code

Reply:
"if you're using your system's terminal" 🙂 

Is the note confusing/unnecessary ? I don't mind removing it.

Reply:
Clarified further for now ("... system terminal outside of the IDE")

Reply:
yes, seems premature and level of details doesn't correlate with the other places in the document

Reply:
Removed.